### ISSUE TITLE
Support for RTX 3060

### ISSUE THREAD

**Issue by vdkhvb** [KnowingNothing/AMOS#3]
When running the example in the ReadME with my RTX 3060 GPU, I keep getting `".E.E.E.E.E"` build error outputs. Iterations never find a proper solution, so the output keeps repeating  things like `"Iteration 2: 1e-10/1e-10(10000000000000.0 ms)"`. Is there something I can do about that? Same thing if I run the test_winograd_cuda file, which gives output like this : 

```
Problem size:
1 256 56 56 512 3 3 1 1
Logging to devnull...
Totally 1 different mappings for this matching
Logging to conv2d-fp16-layer-6-batch-1.log...
Loading from file conv2d-fp16-layer-6-batch-1.log...
Load 0 entries! The best known is 10000000000000.000000 ms
Using arch: sm_86
Total search tirals: 1000 
batch size: 16 
batch num: 63
Search round: 0
.E.E.E.E.E.E.E.E.E.E.E.E.E.E.E.E

Current best timecost:  10000000000000.0 ms
Search round: 1
.E.E.E.E.E.E.E.E.E.E.E.E.E.E.E.E

Current best timecost:  10000000000000.0 ms
Search round: 2
.E.E.E.E.E.E.E.E.E.E.E.E.E.E.E.E

Current best timecost:  10000000000000.0 ms
Search round: 3
.E.E.E.E.E.E.E.E.E.E.E.E.E.E.E.E

Current best timecost:  10000000000000.0 ms
Search round: 4
.E.E.E.E.E.E.E.E.E.E.E.E.E.E.E.E

Current best timecost:  10000000000000.0 ms
Search round: 5
.E.E.E.E.E.E.E.E.E.E.E.E.E.E.E.E

Current best timecost:  10000000000000.0 ms
Search round: 6
.E.E.E.E.E.E.E.E.E.E.E.E.E.E.E.E

Current best timecost:  10000000000000.0 ms
Search round: 7
.E.E.E.E.E.E.E.E.E.E.E.E.E.E.E.E

Current best timecost:  10000000000000.0 ms
Search round: 8
.E.E.E.E.E.E.E.E.E.E.E.E.E.E.E.E

Current best timecost:  10000000000000.0 ms
Search round: 9
.E.E.E.E.E.E.E.E.E.E.E.E.E.E.E.E

Current best timecost:  10000000000000.0 ms
Search round: 10
.E.E.E.E.E.E.E.E.E.E.E.E.E.E.E.E

Current best timecost:  10000000000000.0 ms
Search round: 11
It seems hard to find new candidates...
python-BaseException
Traceback (most recent call last):
  File "/home/vdkhvb/Documents/applications/pycharm/plugins/python-ce/helpers/pydev/pydevd.py", line 1483, in _exec
    pydev_imports.execfile(file, globals, locals)  # execute the script
  File "/home/vdkhvb/Documents/applications/pycharm/plugins/python-ce/helpers/pydev/_pydev_imps/_pydev_execfile.py", line 18, in execfile
    exec(compile(contents+"\n", file, 'exec'), glob, loc)
  File "/home/vdkhvb/Documents/git/AMOS/tests/python/auto_tensorize/test_winograd_cuda.py", line 254, in <module>
    cost = run(
  File "/home/vdkhvb/Documents/git/AMOS/tests/python/auto_tensorize/test_winograd_cuda.py", line 190, in run
    return tensorize_tensorcore_fp16fp16(
  File "/home/vdkhvb/Documents/git/AMOS/tests/python/auto_tensorize/test_winograd_cuda.py", line 166, in tensorize_tensorcore_fp16fp16
    result = at.auto_tensorize_v2(
  File "/home/vdkhvb/Documents/git/AMOS/python/tvm/auto_tensorize/auto_tensorize.py", line 365, in auto_tensorize_v2
    value, params = find_optimized_parameters(
  File "/home/vdkhvb/Documents/git/AMOS/python/tvm/auto_tensorize/search/parameter.py", line 428, in find_optimized_parameters
    params = schedule_gen.get_next(policy=policy)
  File "/home/vdkhvb/Documents/git/AMOS/python/tvm/auto_tensorize/search/parameter.py", line 383, in get_next
    return next(self.gen)
  File "/home/vdkhvb/Documents/git/AMOS/python/tvm/auto_tensorize/search/parameter.py", line 344, in _get_next
    yield self.get(repeat=repeat)
  File "/home/vdkhvb/Documents/git/AMOS/python/tvm/auto_tensorize/search/parameter.py", line 250, in get
    return self.entries[0].record
IndexError: list index out of range

Process finished with exit code 1

```

**Comment by KnowingNothing** [comment]
Here are two steps you may try to help us locate the source of the issue.
1. Try another GPU to see if the problem is specific to 3060.
2. Try to pass verbose flags to `auto_tensorize` interface to see why build fails. For example, 
```py
result = at.auto_tensorize_v4(
        target_dag,
        "cuda",  # code generation target
        "conv2d_tutorial",  # the log file
        measure_opt,
        schedule_log_dir="conv2d_tutorial",
        trials=1200,
        search_group_size=5,
        transform_dump=False,
        verbose=True,
        verbose_schedule=True
    )
```

**Comment by vdkhvb** [comment]
Ah, it was a clang issue with nvcc, errors like these :
```
No supported gcc/g++ host compiler found, but clang-9 is available.
       Use 'nvcc -ccbin clang-9' to use that instead.
```
Ended up updating to Ubuntu 22.04 since apparently 21.10 isn't supported. Reinstalled CUDA 11.7, fixed issues with nvcc not being detected by Python etc. And now the ReadMe test works great.

Anyways, thanks a lot for the quick response. There's no verbose flag in v4 btw, that's in v3. Also, `cd build` is missing in your LLVM installation instructions in the ReadMe, before the cmake call. 
And if it helps others : to build LLVM 10 I had to manually fix a source file (see https://reviews.llvm.org/D89450) + to work with TVM I had to install xgboost==1.5.0 (that specific version).
